authenticator_sdk/src/main/kotlin/com/saltedge/authenticator/sdk/constants/APIConstants.kt

```diff
@@ -59,6 +60,8 @@ const val KEY_CODE = "code"
 const val KEY_CREATED_AT = "created_at"
 const val KEY_UPDATED_AT = "updated_at"
 
+const val KEY_CONNECT_URL_PARAM = "connectUrl"
```

Comment:
remove `_PARAM` suffix

Reply:
we have const with this name but another value

Reply:
it is used by ConnectionsRepository, but this repository is not part of SDK.